WorkAndFun/src/main/java/pl/infoshare/announcements/Categories/ServiceType.java

```diff
@@ -0,0 +1,119 @@
+package pl.infoshare.announcements.Categories;
+
+public enum ServiceType {
+    GLOWNA_KATEGORIA("0", "Głowna kategoria",0),
+    BUDOWA_DOMU("1", "Budowa domu",1),
+    ELEKTRYK("2", "Elektryk",1),
+    HYDRAULIK("3", "Hydraulik",1),
+    MALARZ("4", "Malarz",1),
+    MEBLE_I_ZABUDOWA("5", "Meble i zabudowa",1),
+    MOTORYZACJA("6", "Motoryzacja",1),
+    OGROD("7", "Ogród",1),
+    ORGANIZACJA_IMPREZ("8", "Organizacja imprez",1),
+    PROJEKTOWANIE("9", "Projektowanie",1),
+    REMONT("10", "Remont",1),
+    SPRZATANIE("11", "Sprzątanie",1),
+    SZKOLENIA_I_JEZYKI_OBCE("12", "Szkolenia i języki obce",1),
+    TRANSPORT("13", "Transport",1),
+    USLUGI_DLA_BIZNESU("14", "Usługi dla biznesu",1),
+    MONTAZ_I_NAPRAWA("15", "Montaż i naprawa",1),
+    USLUGI_FINANSOWE("16", "Usługi finansowe",1),
+    USLUGI_PRAWNE_I_ADMINISTRACYJNE("17", "Usługi prawne i administracyjne",1),
+    USLUGI_ZDALNE("18", "Usługi zdalne",1),
+    ZDROWIE_I_URODA("19", "Zdrowie i uroda",1),
+    ZLOTA_RACZKA("20", "Złota rączka",1),
+    INNE("21", "Inne",1),
+
+    BUDOWA_DOMU_BUDOWA_OD_PODSTAW("22", "Budowa od podstaw",2),
```

Comment:
jeśli ktoś doda usługę z kategorii BUDOWA_DOMU_BUDOWA_OD_PODSTAW to w momencie wyswietlenia tego ogloszenia dostaniemy info : KATEGORIA : Budowa od podstaw.

Reply:
po uzgodnieniu zostawiłem jak jest